src/main/java/com/ita/if103java/ims/dao/WarehouseDao.java

```diff
@@ -13,6 +14,10 @@
 
     List<Warehouse> findAll();
 
+    Map<Long, String> findWarehouseNames(Long account_id);
```

Comment:
This method is not used, why did you add it?

Reply:
In ui, I need to have a list of all workers and warehouses in account for filtering, I'm going to use it